usr.bin/elfctl/elfctl.c

```diff
@@ -68,6 +68,8 @@ static struct ControlFeatures featurelist[] = {
 	{ "nostackgap",	NT_FREEBSD_FCTL_STKGAP_DISABLE, "Disable stack gap" },
 	{ "wxneeded",	NT_FREEBSD_FCTL_WXNEEDED, "Requires W+X mappings" },
 	{ "la48",	NT_FREEBSD_FCTL_LA48, "amd64: Limit user VA to 48bit" },
+	{ "cheri_revoke", NT_FREEBSD_FCTL_CHERI_REVOKE_ENABLE, "Enable CHERI revocation" },
```

Comment:
I think the end goal is to default to on and not really need an enable override, but for now I think supporting full symmetry is likely useful. Thinking about this I did review the bit assignment and DISABLE is using one upstream can't reuse (formerly ASG_DISABLE) so that's probably the right decision since that's the knob we'll want to keep.

Reply:
Ok.  I'm kind of thinking that what we probably want ultimately is only a the disable/opt-out `elfctl` flag, only the single flag from `AT_BSDFLAGS`, and only the non-force constants for `malloc_quarantine`.  I'm fine if we have more things in the interim though.

Reply:
I think I've generally structured things so the switch to on-unless-you-opt-out will be fairly clean once we get there. I guess I might want to rethink the MQ_ values.